02_ShowUser.php

```diff
@@ -2,112 +2,110 @@
 <html lang = "eng">
 	<head>
 		<title>Sound Wave</title>
-        <meta charset = "utf-8">
+		<meta charset = "utf-8">
+
+		<!-- styles -->
 		<link rel = "stylesheet" type = "text/css" href = "css/fonts.css">
 		<link rel = "stylesheet" type = "text/css" href = "css/grids.css">
-		<link rel = "stylesheet" href = "css/style.css"/>
```

Comment:
did not need these style sheets anymore; as i already had my other two - grids and fonts